src/bankAccount.test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const BankAccount = require('./bankAccount');
+
+test('Should return account info', () => {
+  const account = new BankAccount('Dmytro', '26.11.1994', 1000);
+
+  expect(account.getInfo())
+    .toBe(`Name: Dmytro, Age: 26, Amount: 1000$`);
+});
+
```

Comment:
Please add a separate test for initial history (before any operarion)

Reply:
Also, will be great to add test for correct constructor
```
const account = new BankAccount('Dmytro', '26.11.1994', 1000);

```
check account is object and and check if account has correct fields